### PR TITLE
Added linux as a module

### PR DESCRIPTION
**Pull request by AdityaAtulTewari**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[submodule "module/linux"]
+	path = module/linux
+	url = git@github.com:utcs-scea/linux.git
```
Comment (patrickkenney9801) [inline .gitmodules:3]: this is required to ignore the symlink defined below?

Comment (AdityaAtulTewari) [reply .gitmodules:3]: No that was accidentally kept in let me fix that.

**Review by patrickkenney9801 — APPROVED** [review]
Great

**Comment by AdityaAtulTewari** [comment]
Before I merge running a fresh repo to make sure nothing breaks

**Comment by AdityaAtulTewari** [comment]
@patrickkenney9801 do we want to add a warning in the readme from cloning all submodules?

**Comment by patrickkenney9801** [comment]
> @patrickkenney9801 do we want to add a warning in the readme from cloning all submodules?

yes:

`WARNING: git clone ... --recurse-submodules may result in regret faster than a speeding bullet and pain greater than getting shot`

**Pull request merged**